src/cc2olx/link_file_reader.py

```diff
@@ -0,0 +1,52 @@
+import csv
+
+
+class LinkFileReader:
+    """
+    This class is responsible to read the csv file that is provided and generates
+    a map of link to the row where this link belongs to.
+
+    +------------------------------------------------------+--------------------------------------+-------------+
+    |                 External Video Link                  |                Edx Id                | Youtube Id  |
+    +------------------------------------------------------+--------------------------------------+-------------+
+    | https://cdnapisec.kaltura.com/p/2019031/sp/201903100 | 42d2a5e2-bced-45d6-b8dc-2f5901c9fdd0 | onRUvL2SBG8 |
```

Comment:
Nit: this example url doesn't include `entry_id` and leads to `KeyError` [here](https://github.com/edx/cc2olx/blob/9c407b1e6ab3f161b9f9754de9cd25bd8b01e564/src/cc2olx/iframe_link_parser.py#L151).

Reply:
updated thanks